### ISSUE TITLE
[EPIC] Transaction settings for send flow: Nonce and Gas price

### ISSUE THREAD

**Issue by vkjr** [status-im/status-react#21016]
### User story

As a wallet user,
I want to edit the parameters of my transaction, like gas, nonce, etc
So I can control how my txn is broadcasted to the network

### Description

Currently, the txn is sent with default computed parameters. This is not ideal as the user might want to speed up or cancel txns, by changing nonce and gas price. 

This epic implements advanced txn settings for send flow. The same might apply to swaps and smart contract interactions, but for this version, we only build for send flow. 

Figma Link:  https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17669-40876
Schedule for 2.32 https://www.notion.so/Status-Apps-Planning-Releases-11e8f96fb65c801289c5e37099c4039b

## Tech debt
- [x] #21017

## Notes
- Status Go PR: https://github.com/status-im/status-go/pull/6124

**Comment by vkjr** [comment]
@shivekkhurana, @smohamedjavid, fyi.
I've created a basic set of issues to implement transaction settings discussed in the meeting. Every issue contains the bare minimum and should be refined later.

**Comment by vkjr** [comment]
Blocked until designs are finished by @xAlisher 
[Figma drafts (not for development)](https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17669-40876&t=TK3SuJ90iMM5VoXX-1)

**Comment by shivekkhurana** [comment]
@vkjr @alwx We have this scheduled for 2.32. Although the designs are not final, the UX has been agreed upon. 

We can start working on this. If there are UI changes requested, we can add it later.

**Comment by alwx** [comment]
@shivekkhurana I've started working on this: https://github.com/status-im/status-mobile/issues/21551

**Comment by churik** [comment]
I do not think that it is feasible to include this feature to  2.32, postponing due to limited capacity in order to keep release scope clear

cc @shivekkhurana

**Comment by churik** [comment]
afaik Nonce and as Price is not feasible to include to 2.33, postponing for now cc @shivekkhurana - re-app if still needs to be included

**Comment by mohsen-ghafouri** [comment]
We can close this @shivekkhurana

**Comment by vkjr** [comment]
@mohsen-ghafouri, I believe we don't have a way to set custom transaction parameters for approval transactions, don't we? If so, probably this is something worth adding to the epic. @shivekkhurana, wdyt?

**Comment by mohsen-ghafouri** [comment]
Yes there wasn't any request to add transaction setting button for swap approval. If needed i can add it.
